packages/react/src/drafts/IssueLabel/IssueLabel.tsx

```diff
@@ -43,8 +45,17 @@ export interface IssueLabelProps extends React.PropsWithChildren {
 }
 
 export function IssueLabel({children, fillColor, size = 'small', variant = 'gray', ...rest}: IssueLabelProps) {
+  const {resolvedColorScheme, theme} = useTheme()
+  // TODO: get the bgColor, getting it from theme.colorScheme seems a bit sketchy
+  console.log('theme', theme)
+
   return (
-    <StyledLabel {...rest} data-size={size} data-variant={fillColor ? undefined : variant}>
+    <StyledLabel
+      {...rest}
+      data-size={size}
+      data-variant={fillColor ? undefined : variant}
+      style={fillColor ? getColorsFromHex(fillColor, resolvedColorScheme, '#fff') : undefined}
+    >
```

Comment:
@joshblack I need to get the default backgroundColor `bgColor-default` for the current selected theme. How would you suggest I bet retrieve it?

Reply:
@lukasoppermann most likely the best way is what you have right now which is `useTheme()`. I think that no matter what we'll need to use a context provider to pipe this through since we can't read this value from anywhere during SSR. One thing I'm not sure about is how teams are setting this so that the correct theme is set during SSR so that we can avoid a flash between themes.

Reply:
Okay. One _issue_ I am seeing is that the color is using a css variable. At the moment we still have a fallback value, but I need a hex value to calculate this. 

Do you think I should create an object with the background colors in here, like this

```js
const bgColors = {
  light: "#ffffff",
  dark: "#0d1117"
}
```

The issue I see here is that it is easy to fall out of sync with primitives if they get changed.

Do you have any better idea? I guess I could use just the two dark and light values above for all themes and if they change it would probably not be that much of a problem.

Reply:
@joshblack I implemented my idea from above. Is there anything else I should change?

Reply:
Looks great 👍 The only thought was that it would be great if we could pull the values themselves from `@primer/primitives` but that would be outside of the scope of this PR.

Reply:
Yes, I'd love to do this. Realistically the difference should not be so big, so if this has a speed impact we should not do it.